setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -u
+# ======= COPY FILES =========
+echo "Setting up essential files"
+fls=(envs, .bashrc, .bash_profile, .projects, .nodeload)
+for fl in ${fls[@]}; do
+  if [ -f ${fl} ]; then
+    mv $(realpath ${fl}) ~/
+  fi
+done
+
+# ======= SETUP MINICONDA =========
+echo "Starting Miniconda setup"
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+CONDAPATH=/om2/user/$(whoami)/miniconda
```

Comment:
We should check whether this directory exists before installing. If it exists, we should not install miniconda.